Makefile

```diff
@@ -198,7 +198,7 @@ build-kustomize-api: $(builtinplugins)
 
 .PHONY: test-unit-kustomize-api
 test-unit-kustomize-api: build-kustomize-api
-	cd api; go test ./...
+	cd api; go test ./...  -ldflags "-X sigs.k8s.io/kustomize/api/provenance.version=v1"
```

Comment:
please make this version something obviously strange but complete with major.minor.patch, e.g.
`=v444.333.222`
That way one can grep for it and match the makefile target to the test.
other than that,looks fine.

Reply:
Updated it to `v444.333.222`.